src/sphinxcontrib/towncrier/_scm_version.pyi

```diff
@@ -0,0 +1 @@
+version: str
```

Comment:
I'd also add a note that this is necessary because the actual module is autogenerated on build and may be missing at the time `mypy` runs.